packages/react-magma-dom/src/components/TreeView/TreeItem.test.js

```diff
@@ -211,4 +211,38 @@ describe('TreeItem', () => {
       expect(treeItem).toHaveFocus();
     });
   });
+  describe('hover styles', () => {
+    it('should have default hover color', () => {
+      const { getByTestId } = render(
+        <TreeItem label={labelText} testId={testId} itemId={itemId} />
+      );
+
+      expect(getByTestId(testId)).toBeInTheDocument();
+      expect(getByTestId(testId)).toHaveStyleRule(
+        'background',
+        transparentize(0.95, magma.colors.neutral900),
+        {
+          target: ':hover',
+        }
+      );
+    });
+
+    it('should have custom hover color', () => {
+      const { getByTestId } = render(
+        <TreeItem
+          label={labelText}
+          testId={testId}
+          itemId={itemId}
+          hoverColor={magma.colors.blue500}
```

Comment:
blue500 is not a value in our theme
```suggestion
          hoverColor={magma.colors.primary500}
```

Reply:
Ops, typo!